oh_app_demo/urls.py

```diff
@@ -24,3 +33,7 @@
     path('', include('main.urls')),
     path('admin/', admin.site.urls),
 ]
+
+urlpatterns += [
+    url('', include(('openhumans.urls', 'openhumans'), namespace='openhumans'))
```

Comment:
I'd prefer to be using a prefix for included app URLs, e.g.
`url('openhumans/', include('openhumans.urls'))`